capabilities/capability-service.yaml

```diff
@@ -307,8 +307,6 @@ components:
         - $ref: '#/components/schemas/Dimension'
         - type: object
           properties:
-            type:
```

Comment:
`type` is still on here (from `Dimension`)

Reply:
Good spot - should be removed now